src/modules/esl-toggleable/README.md

```diff
@@ -1,8 +1,6 @@
 # [ESL](../../../README.md) Toggleable
 
-Version: *1.0.0-beta*.
-
-***Important Notice: the component is under beta version, it was tested and now ready to use but be aware of its potential critical API changes.***
+Version: *1.0.0*.
```

Comment:
The point at the end of sentences. Either delete it here or return it to the rest.